lib/matplotlib/textpath.py

```diff
@@ -99,7 +99,7 @@ def get_text_path(self, prop, s, ismath=False):
             from matplotlib.text import TextToPath
             from matplotlib.font_manager import FontProperties
 
-            fp = FontProperties(family="Humor Sans", style="italic")
+            fp = FontProperties(family="comic neue", style="italic")
```

Comment:
```suggestion
            fp = FontProperties(family="Comic Neue", style="italic")
```
Not sure if this is case sensitive (probably not), but for consistency with the rest.